resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+# Dedicated cluster role - istiod will use fewer dangerous permissions ( secret access in particular ).
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: istiod-internal-{{ .Values.revision | default "default" }}-{{ .Release.Namespace }}
```

Comment:
why add this?

Reply:
I'm asking because there's already a ClusterRole `istiod-default-{namespace}`

Reply:
The revision stuff is how multiple control planes are supported, so this ensures a unique name.  Same comment applies to every place else where the name went from <name>-<namespace> to <name>-<revision>-<namespace>.  (Writing this, I'm wondering if there are other places where revision needs to be added, e.g. mixer, prometheus, kiali, etc.)